### ISSUE TITLE
[Bug]: Sysav_se stoped working (working localy)

### ISSUE THREAD

**Issue by MHultman** [mampfes/hacs_waste_collection_schedule#1576]
### I Have A Problem With:

A specific source

### What's Your Problem

I am the creator of Sysav_se source and can't get it to work properly again. 
All working localy when i extract the code inside the source. 

As far as I understand it is the call for checking the address that fails, and I can't seem to figure out why.

### Source (if relevant)

sysav_se

### Logs

```Shell
fetch failed for source Sysav Sophämntning: 
Traceback (most recent call last): 
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch
    entries = self._source.fetch() 
    # ^^^^^^^^^^^^^^^^^^^^ 
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/sysav_se.py", line 29, in fetch
    address_data = json.loads(response.text) 
    # ^^^^^^^^^^^^^^^^^^^^^^^^^ 
  File "/usr/local/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s) 
    # ^^^^^^^^^^^^^^^^^^^^^^^^^^ 
  File "/usr/local/lib/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end()) 
    # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
  File "/usr/local/lib/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None 
    # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
    - name: sysav_se
      args:
        street_address: Sommargatan 1, Svedala

sensor: 
  - platform: waste_collection_schedule
    name: sophamtning
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
Duplicate of #1536

**Comment by 5ila5** [comment]
>Fix will be part of release 1.45.0 or is available now as master (git) version (HACS: 3-dot menu -> redownload -> change version to master)
